### ISSUE TITLE
Jest API Reference pages

### ISSUE THREAD

**Issue by tjmehta** [epitaphmike/jest-dash#2]
It would be nice to have the [API Reference](https://facebook.github.io/jest/docs/api.html) pages searchable in Dash. When using this docset in Dash I only see guides.. am I missing something?

**Comment by alexgurr** [comment]
I agree with tjmehta. This seems useful, but for me the API docs are the more important bit I need to reference. Would love to see those in here!
